components/Footer.js

```diff
@@ -0,0 +1,58 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import styled from 'styled-components'
+import {
+  faFacebook,
+  faInstagram,
+  faLinkedinIn,
+  faMediumM,
+  faTwitter,
+  faYoutube,
+} from '@fortawesome/free-brands-svg-icons'
+
+const FooterContainer = styled.div`
+  background: #3A396E;
+  color: #20FFAF;
```

Comment:
This might be out of scope for this PR but wondering if we should set up [ThemeProvider](https://styled-components.com/docs/advanced#theming) to define commonly used colours like these ones? Happy to set this up for the repeating styles we're using.

Reply:
Ohhh yes, that would be great! I don't have experience with using ThemeProvider so I'll count on you for that - good call

Reply:
Ok! opened #20